documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -58,214 +53,110 @@ from additional images.
 
-A `serverPod` can be defined at the domain scope, which applies to every pod in
-the domain, plus the introspector job's pod, at a specific WebLogic cluster's scope,
-or at a specific WebLogic Server pod's scope.
-Typically, the domain scope is
-the most applicable for the Model in Image use case; for example:
+See [Configuration Example 1: Basic Configuration](#configuration-example-1-basic-configuration).
 
-```
-spec:
-  serverPod:
-    auxiliaryImages:
-    - image: model-in-image:v1
-      imagePullPolicy: IfNotPresent
-      volume: auxiliaryImageVolume1
-```
+#### Source Locations
+Use the optional attributes `sourceModelHome` and `sourceWdtInstallHome` to specify non-default locations of 
+WebLogic Deploy Tooling model and installation files in your auxiliary image(s).
+- Allowed values for `sourceModelHome` and `sourceWdtInstallHome`:
+  - Unset - Default to `/auxiliary/models` and `/auxiliary/weblogic-deploy`, respectively.
+  - Set to a path - Must point to an existing location containing WDT model and installation files, respectively.
+  - `None` - Indicates that the image has no WDT models and installation files, respectively. 
 
-{{% notice note %}}
-If image pull secrets are required for pulling auxiliary images,
-then the secrets must be referenced using `domain.spec.imagePullSecrets`.
-{{% /notice %}}
+If you set the `sourceModelHome` or `sourceWDTInstallHome` to `None` or, there are no files at the default locations 
+(`/auxiliary/models` and `/auxiliary/weblogic-deploy`), then the operator will ignore the source directories.
+Note that if you set a source directory value and there are no files in that directory in the auxiliary image, 
+then the domain deployment will fail.
 
-#### Model in Image paths
+The files in `sourceModelHome` and `sourceWDTInstallHome` directories will be made available in `/aux/models` 
+and `/aux/weblogic-deploy` directories of the WebLogic Server container in all pods, respectively.
 
-For the Model In Image auxiliary image use case, you also need to
-configure the `domain.spec.configuration.model.modelHome`
-and `domain.spec.configuration.model.wdtInstallHome` attributes
-to specify the location of the domain's WebLogic Deploy Tool (WDT)
-model files and the domain's WDT installation.
-These default to `/u01/wdt/models` and `/u01/wdt/weblogic-deploy`
-respectively, and must be changed to specify a directory in
-`domain.spec.auxiliaryImageVolumes.mountPath`. For example:
+See [Configuration Example 2: Source Locations](#configuration-example-2-source-locations).
 
-```
-  configuration:
-    model:
-      modelHome: "/auxiliary/models"
-      wdtInstallHome: "/auxiliary/weblogic-deploy"
-```
+#### Multiple Auxiliary Images
+If specifying multiple auxiliary images with model files in their respective `sourceModelHome` directories, then model files are merged. 
```

Comment:
sourceModelHome --> configuration.model.auxiliaryImages[].sourceModelHome
ditto for sourceWdtInstallHome

(so that it's obvious these two values are part ofthe aux image stanza)